lib/api/v3/utilities/custom_field_injector.rb

```diff
@@ -303,7 +303,10 @@ def allowed_users_href_callback
                                  represented.project_id.to_s
                                end
 
-            filters = static_filters << { member: { operator: '=', values: [project_id_value.to_s] } }
+            # Dup is important here to avoid appending to the object
+            # made available in the closure (which is class level) which would
+            # keep the appended filters between requests.
+            filters = static_filters.dup << { member: { operator: '=', values: [project_id_value.to_s] } }
```

Comment:
Do we need `.dup` here? I would simply write this as `filters = static_filters + { member: { operator: '=', values: [project_id_value.to_s] } }`

Reply:
fine with me as well